kore/src/Kore/Attribute/Axiom.hs

```diff
@@ -153,7 +163,52 @@ instance Default Axiom where
             , owise = def
             }
 
-instance ParseAttributes Axiom where
+instance ParseAttributes (Axiom SymbolOrAlias) where
+    parseAttribute attr =
+        typed @HeatCool (parseAttribute attr)
+        Monad.>=> typed @ProductionID (parseAttribute attr)
+        Monad.>=> typed @Priority (parseAttribute attr)
+        Monad.>=> typed @Assoc (parseAttribute attr)
+        Monad.>=> typed @Comm (parseAttribute attr)
+        Monad.>=> typed @Unit (parseAttribute attr)
+        Monad.>=> typed @Idem (parseAttribute attr)
+        Monad.>=> typed @Trusted (parseAttribute attr)
+        Monad.>=> typed @Concrete (parseAttribute attr)
+        Monad.>=> typed @Simplification (parseAttribute attr)
+        Monad.>=> typed @(Overload SymbolOrAlias) (parseAttribute attr)
+        Monad.>=> typed @SmtLemma (parseAttribute attr)
+        Monad.>=> typed @Label (parseAttribute attr)
+        Monad.>=> typed @SourceLocation (parseAttribute attr)
+        Monad.>=> typed @Constructor (parseAttribute attr)
+        Monad.>=> typed @Functional (parseAttribute attr)
+        Monad.>=> typed @Subsorts (parseAttribute attr)
+        Monad.>=> typed @UniqueId (parseAttribute attr)
+        Monad.>=> typed @Owise (parseAttribute attr)
+
+    toAttributes =
+        mconcat . sequence
+            [ toAttributes . heatCool
+            , toAttributes . productionID
+            , toAttributes . priority
+            , toAttributes . assoc
+            , toAttributes . comm
+            , toAttributes . unit
+            , toAttributes . idem
+            , toAttributes . trusted
+            , toAttributes . concrete
+            , toAttributes . simplification
+            , toAttributes . overload
+            , toAttributes . smtLemma
+            , toAttributes . label
+            , toAttributes . sourceLocation
+            , toAttributes . constructor
+            , toAttributes . functional
+            , toAttributes . subsorts
+            , toAttributes . uniqueId
+            , toAttributes . owise
+            ]
+
+instance ParseAttributes (Axiom Symbol) where
```

Comment:
Because it's not possible to parse an `Overload Symbol` attribute, it's also not possible to parse an `Axiom Symbol` attribute.

Reply:
As I remember, after I changed `RulePattern` 's `attributes` to `Attribute.Axiom Symbol` , the compiler requested a `SQL.Column` instance for  `Axiom Symbol`. I just kept adding the required instances until I did this. I'll try to do it in a different way

Reply:
Oh, it wants the `toAttributes` function. That's the only one you should need to implement. You can leave the other methods as `undefined`, and I will push a separate pull request which deals with that; I already have it in a branch.

Reply:
but without
`parseAttribute  _ (Overload Nothing) = return $ Overload Nothing`
4 tests are failing.
Should I just use `undfined` and wait for your pull request?

Reply:
Let's just merge #1510 quickly, then this.